docker/Dockerfile

```diff
@@ -1,15 +1,14 @@
 FROM jupyter/datascience-notebook:latest
-MAINTAINER Serenata de Amor "datasciencebr@gmail.com"
 
 USER root
 
-RUN apt-get update && apt-get install -y \
-  unzip
+RUN apt-get update && apt-get install -y unzip
 
 USER jovyan
 
 COPY requirements.txt ./
 COPY conda_requirements.txt ./
+COPY config.ini.example ./config.ini
```

Comment:
Excuse me for jumping in out of nowhere but this doesn't matter when it comes to local development environments since the whole dir will be overwritten with the project code because of [this](https://github.com/datasciencebr/serenata-de-amor/blob/0a338f09406834c372786984d1a3fe5d6e0383f8/docker-compose.yml#L11). HTH 😄